content/docs/hooks-overview.md

```diff
@@ -128,9 +128,9 @@ function FriendStatus(props) {
 }
 ```
 
-In this example, React would unsubscribe from our `ChatAPI` when the component unmounts, as well as before re-running the effect due to a subsequent render. (If you want, there's a way to [tell React to skip re-subscribing](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) if the `props.friend.id` we passed to `ChatAPI` didn’t change.)
+इस उदाहरण में, React हमारे `ChatAPI` से तब अनसब्सक्राइब हो जाती है जब कौम्पोनॅन्ट अनमाउंट हो जाता है, साथ ही बाद में रेंडर करने के कारण effect को फिर से चलाने से पहले। (यदि आप चाहते हैं, यहाँ एक तरीका है [React को री-सब्सक्राइब छोड़ना बताएं।](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) अगर 'props.friend.id' जो हमने `ChatAPI` को पास किया है,नहीं बदलता तो।)
```

Comment:
`[React को री-सब्सक्राइब छोड़ना बताएं।](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects)` => `[React को री-सुब्स्क्रिबिंग को स्किप करने के लिए बताने का](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects)`

Reply:
`री-सुब्स्क्रिबिंग` looks wrong... `सब्सक्राइबिंग` looks good?

Reply:
`React को री-सुब्स्क्रिबिंग को स्किप करने के लिए बताने का`
 I don't think this sentence is correct, it doesn't go well with what we have to tell. What do you think?

Reply:
`[React को बताने का की री-सब्सक्राइबिंग को कैसे स्किप करें](/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects) अगर `props.friend.id` ना बदले जो `ChatAPI` को पास किया है`

how about this? or do you have any other suggestions?

Reply:
`बताने का` is sounds wrong, I would say: `बताये`. Rest looks good.

Reply:
> `बताने का` sounds wrong, I would say: `बताये`. Rest looks good.

if you read the whole sentence `बताने का` will make more sense
`यहाँ एक तरीका है [React को री-सब्सक्राइब छोड़ना बताएं।]`